### PR TITLE
[Site] add import maps button and more query string

### PR DESCRIPTION
**Pull request by kavilla**
### Description

Enables access to easily see the import custom maps. Also added more query strings to easily set and build a link to the test results page to be easily copied and shared.

Signed-off-by: Kawika Avilla <kavilla414@gmail.com>

### Issues Resolved

n/a

### Check List

- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by CCongWang — APPROVED** [review]

File: site/js/dashboard.js [hunk 1]
```diff
@@ -131,6 +140,20 @@
...
+    `version=${version}&` +
+    `build_number=${buildNumber}&` +
+    `test_number=${testNumber}&` +
+    `platform=${platform}&` +
+    `arch=${arch}&` +
+    `type=${type}`;
+
+  var resultPageLink = document.getElementById('resultPageLink');
+  resultPageLink.textContent = resultPageUrl;
+  resultPageLink.href = resultPageUrl;
```
[the remark this answers is not in the document]
Comment (Flyingliuhub) [reply site/js/dashboard.js:155]: are you planning to fix "DOM text reinterpreted as HTML"? Thanks

**Comment by Flyingliuhub** [comment]
Is failed cypress test known issue? https://github.com/opensearch-project/opensearch-dashboards-functional-test/actions/runs/3127002289/jobs/5073142397

**Review by Flyingliuhub — APPROVED** [review]

**Review by tianleh — APPROVED** [review]

File: site/js/dashboard.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (tianleh) [reply site/js/dashboard.js:155]: Since this is `Medium`, we can tackle later.

**Comment by tianleh** [comment]
> @kavilla , Is failed cypress test known issue? https://github.com/opensearch-project/opensearch-dashboards-functional-test/actions/runs/3127002289/jobs/5073142397

This is unrelated to @kavilla 's change.

File: site/js/dashboard.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (kavilla) [reply site/js/dashboard.js:155]: Created an issue for this: https://github.com/opensearch-project/opensearch-dashboards-functional-test/issues/294

**Pull request merged**